independent-publisher-connectors/AzureFunctionApps/apiDefenition.swagger.json

```diff
@@ -0,0 +1,89 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Default title",
+        "description": "",
+        "version": "1.0"
+    },
+    "host": "[YOURFUNCTIONAPP].azurewebsites.net",
+    "basePath": "/",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [],
+    "produces": [],
+    "paths": {
+        "/api/DynamicsAuthStart": {},
+        "/": {},
+        "/DynamicsAuth": {},
+        "/api/DynamicsAuth": {
+            "post": {
+                "responses": {
+                    "default": {
```

Comment:
@Stephan-Rijk it is not allowed to have a default response only. please define a proper response. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).